user_tools/src/spark_rapids_pytools/cloud_api/emr.py

```diff
@@ -272,6 +273,7 @@ def get_instance_description_cli_params(self):
     def init_instance_descriptions(self) -> None:
         platform = CspEnv.pretty_print(self.cloud_ctxt['platformType'])
         instance_description_file_path = Utils.resource_path(f'{platform}-instance-catalog.json')
+        self.logger.info(f'Loading instance descriptiond from cached file: {instance_description_file_path}')
```

Comment:
nit: descriptiond -> description

Reply:
Thanks @nartal1! I think I fixed this. Could you try refreshing the page?

Reply:
Thanks @cindyyuanjiang ! I thought this was the latest  commit https://github.com/NVIDIA/spark-rapids-tools/pull/1196/commits/c6aa9225679b245df8dd5c8c884743c596e12f3d and was reviewing based on that.